src/inputs/Integer.jsx

```diff
@@ -7,7 +8,7 @@ const IntegerInput = ({ value, name, label, title, setValue, validator = () => t
     setValue(e.target.name, value);
   }
   return (
-      <div>
+      <React.Fragment>
```

Comment:
oooh I actually style this in my open PR haha. I see what you mean now. I do think semantically it makes sense as a <div class=“input-group”/> or something like that